src/pipecat/services/google/llm_openai.py

```diff
@@ -40,6 +40,10 @@ class GoogleLLMOpenAIBetaService(OpenAILLMService):
     Note: This service includes a workaround for a Google API bug where function
     call indices may be incorrectly set to None, resulting in empty function names.
 
+    .. deprecated:: 0.0.81
```

Comment:
Note to self: I'm assuming this PR will make it in for 0.0.81. Otherwise, update this!

Reply:
Update to 0.0.82. This'll make the next release.